<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>412</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>光网络仿真</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/img/狗狗.png</normaloff>:/img/狗狗.png</iconset>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWebEngineView" name="webWidget" native="true">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>548</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>项目</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveProject"/>
    <addaction name="separator"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionAdd_node"/>
    <addaction name="actionAdd_link"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>构建</string>
    </property>
    <addaction name="actionmap"/>
    <addaction name="separator"/>
    <addaction name="actionConfig"/>
    <addaction name="actionPlay"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNodes_view"/>
    <addaction name="actionLinks_view"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonFollowStyle</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionexit"/>
   <addaction name="separator"/>
   <addaction name="actionOpenProject"/>
   <addaction name="actionSaveProject"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_node"/>
   <addaction name="actionAdd_link"/>
   <addaction name="separator"/>
   <addaction name="actionmap"/>
   <addaction name="actionPlay"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidgetNode">
   <property name="windowIcon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/Core switch.png</normaloff>:/img/Core switch.png</iconset>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>节点视图</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="treeWidgetNode">
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>false</bool>
       </property>
       <column>
        <property name="text">
         <string notr="true">交换节点</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>坐标X</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>坐标Y</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>删除</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetLink">
   <property name="windowIcon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/optical_fiber.png</normaloff>:/img/optical_fiber.png</iconset>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>链路视图</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="treeWidgetLink">
       <property name="autoExpandDelay">
        <number>-1</number>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="itemsExpandable">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
       <property name="allColumnsShowFocus">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="headerHidden">
        <bool>false</bool>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>false</bool>
       </property>
       <attribute name="headerCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="headerHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="headerShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>链路</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>起点</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>终点</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>长度</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>光纤数量</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>更多</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>删除</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNewProject">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/file-add.png</normaloff>:/img/file-add.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Shift+N</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Shift+S</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/file-open.png</normaloff>:/img/file-open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Shift+O</string>
   </property>
  </action>
  <action name="actionAdd_node">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/node.png</normaloff>:/img/node.png</iconset>
   </property>
   <property name="text">
    <string>添加节点</string>
   </property>
   <property name="shortcut">
    <string>Shift+N</string>
   </property>
  </action>
  <action name="actionAdd_link">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/link.png</normaloff>:/img/link.png</iconset>
   </property>
   <property name="text">
    <string>添加链路</string>
   </property>
   <property name="shortcut">
    <string>Shift+L</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/play.png</normaloff>:/img/play.png</iconset>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
   <property name="shortcut">
    <string>Shift+P</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/design.png</normaloff>:/img/design.png</iconset>
   </property>
   <property name="text">
    <string>仿真配置</string>
   </property>
   <property name="toolTip">
    <string>仿真配置</string>
   </property>
   <property name="shortcut">
    <string>Shift+C</string>
   </property>
  </action>
  <action name="actionNodes_view">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/exchange_node.png</normaloff>:/img/exchange_node.png</iconset>
   </property>
   <property name="text">
    <string>节点视图</string>
   </property>
  </action>
  <action name="actionLinks_view">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/optical_fiber.png</normaloff>:/img/optical_fiber.png</iconset>
   </property>
   <property name="text">
    <string>链路视图</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/exit.png</normaloff>:/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionmap">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/map.png</normaloff>:/img/map.png</iconset>
   </property>
   <property name="text">
    <string>虚拟化映射</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
